### PR TITLE
Explain how to securely transfer school check data

### PR DESCRIPTION
**Pull request by lawrence-forooghian**

### CODE REVIEW

**Comment by lawrence-forooghian** [comment]
[Export school checks data for service team](https://trello.com/c/zpnaNlrB/1406-export-school-checks-data-for-service-team)

File: docs/school-check-data.md [hunk 1]
```diff
@@ -52,10 +52,15 @@ which need this email sending is a developer task, documented below.
    `school-check-batch_ddmmyyyy.csv`
 
 8. Securely transfer the file to the relevant service operator to follow the
-   rest of the school check process
+   rest of the school check process. The best way we’ve found to do this is to
+   create a new Google Sheets document, private to yourself and shared only with
```
Comment (jacksonj04) [inline docs/school-check-data.md]:
```suggestion
   create a [new Google Sheets document](https://sheets.new/), private to yourself and shared only with
```

Comment (lawrence-forooghian) [reply docs/school-check-data.md]: Is this documented by Google somewhere?

Comment (lawrence-forooghian) [reply docs/school-check-data.md]: Found this: https://twitter.com/googledocs/status/1055490445088903168

Comment (jacksonj04) [reply docs/school-check-data.md]: https://support.google.com/a/users/answer/9308871?hl=en has an overview.

**Pull request merged**